app/src/views/private/components/flow-sidebar-detail.vue

```diff
@@ -192,6 +192,11 @@ const runManualFlow = async (flowId: string) => {
 					v-tooltip="getFlowTooltip(manualFlow)"
 					small
 					full-width
+					:style="{
+						'--v-button-background-color': manualFlow.color,
+						'--v-button-background-color-hover': manualFlow.color,
+						'--v-button-background-color-active': manualFlow.color,
+					}"
```

Comment:
Since it is not being used in this case, there is no need to set an active color. In fact, we only need to set the default background color. I added a commit for the hover color, which mixes the color with light and dark shades for consistency.

```suggestion
					:style="{ '--v-button-background-color': manualFlow.color }"
```